sqlalchemy_ibmi/base.py

```diff
@@ -762,6 +763,11 @@ def _get_default_schema_name(self, connection):
             default_schema_name = default_schema_name.strip()
         return self.normalize_name(default_schema_name)
 
+    def get_driver_version(self, db_conn):
+        version = db_conn.getinfo(self.dbapi.SQL_DRIVER_VER).split('.')
+        mmsp = version.pop(2)
+        return list(map(int, version + [mmsp[:2], mmsp[2:]]))
```

Comment:
You'll also need to handle when the service pack is 0, since it won't actually be set, ie. 07.01.00 instead of 07.01.000. As it currently is, in that case the second slice will be an empty string, which will raise a ValueError when passed to `int()`.

Reply:
I'd recommend doing the pop, then split the popped argument (the slicing is fine here) in to a new array and replace the empty string (if found) with a 0 or '0', then join these arrays (or extend the original array with the new arrays) and pass it to the list comprehension.